packages/vuetify/src/components/VForm/__tests__/VForm.spec.ts

```diff
@@ -1,28 +1,49 @@
+// Libraries
 import Vue from 'vue'
+import Vuetify from '../../../framework'
+
+// Components
+import VForm from '../VForm'
+import VTextField from '../../VTextField'
+
+// Utilties
 import {
   mount,
   MountOptions,
   Wrapper,
 } from '@vue/test-utils'
-import VTextField from '../../VTextField'
-import VBtn from '../../VBtn'
-import VForm from '../VForm'
+
 import { wait } from '../../../../test'
 
-const inputOne = Vue.component('input-one', {
-  render (h) {
-    return h(VTextField, {
-      props: [v => !!v || 'Required'],
-    })
-  },
-})
+Vue.use(Vuetify)
```

Comment:
What is the purpose of this?

Reply:
It came in via the dev branch merge. The wait function is used on line 178.